package.json

```diff
@@ -7,11 +7,14 @@
   "scripts": {
     "dev": "nodemon .",
     "start": "node .",
-    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest"
+    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest",
+    "sass-dev": "sass --watch --update --style=expanded static/src/main.scss:static/assets/main.css",
+    "sass-prod": "sass --no-source-map --style=compressed static/src/main.scss:static/assets/main.css"
```

Comment:
I'm not sure where the "build" command has gone that was previously there, but I think you need to change the script name for running sass compiler to "build" since it says so in the instructions

Reply:
changed to the correct script now